apps/auction2/components/auction_grid/index.js

```diff
@@ -0,0 +1,18 @@
+import { default as React, PropTypes } from 'react';
+import AuctionGridArtwork from '../auction_grid_artwork/index'
+
+export default function AuctionGrid({ artworks }, _) {
```

Comment:
I actually kind of like `PropTypes` in this case since it makes it very obvious which params the view requires, but we can drop it too if people find it unwieldy.